_layouts/default.html

```diff
@@ -4,15 +4,15 @@
 {% include head.html %}
 
 <body>
-	<main class="container">
-		<header>
-			{% include navbar.html %}
-		</header>
-		<section class="content">
-			{{ content }}
-		</section>
-	{% include footer.html %}
-	</main>
+		<main class="container">
+				<header>
+						{% include navbar.html %}
```

Comment:
I don't know.  I was trying to follow your suggestion of 4 spaces for HTML, so I added extra space.  Not sure why it was interpreted as so much extra space.  Will attempt a fix.